docs/extensibility/resources.md

```diff
@@ -0,0 +1,226 @@
+
+The Config Map needs to have several fields created that define how the resources is handled.
+
+All sections can be provided as either json or yaml.
+
+## Config Map sections
+
+### `resource` section
+
+The resource section is required and contains basic information about the resource - namely kind and api details.
+
+#### Example
+
+```json
+{
+  "kind": "MyResources",
+  "group": "networking.istio.io",
+  "version": "v1alpha3"
+}
+```
+
+### `navigation` section
+
+The navigation section is required and defines this resource's availability in the menu and generated url. The name of the menu entry as well as category it's going to be placed in are read from the translation section's `name` and `category` entries respectively.
+
+#### Item paremeters
+
+- **path** - path for this resource to use in the url.
+- **scope** - either `namespace` or `cluster`.
+
+#### Example
+
+```json
+{
+  "path": "my-resources",
+  "scope": "namespace"
+}
+```
+
+### `form` section
+
+Form section modifies the original schema (provided in the schema section) for use with [ui-schema](https://ui-schema.bemit.codes/) to generate the create/edit form for the resource. It contains a list of objects that define what fields should be included in the final form. All given fields are placed in the advanced form by default, however it's possible to add a field to simple form by providing the `simple: true` flag, or removing it from advanced form by providing the `advanced: false` flag.
+
+#### Item parameters
+
+- **path** - _[required]_ path to the property that should be displayed in the form. In case of an array the array index is ommited (so for instance if `spec.items` is an array and we want to display `name` for each items, the path would simply be `spec.items.name`).
+- **widget** - optional widget to render the field. If no widget is provided a default handler will be used depending on the data type provided in the schema. For more information about the available widgets see [Form widgets](form-widgets.md).
+- **simple** - whether to display in the simple form. By default it is false.
+- **advanced** - whether to diplay in the advanced form. By default it is true.
+
+### Example
+
+```json
+[
+  { "path": "spec.priority", "simple": true },
+  { "path": "spec.items.name" },
+  { "path": "spec.items.service.url" },
+  { "path": "spec.items.service.port" }
+]
+```
+
+### `list` section
+
+List section defines extra columns available in the list. The format is similar to the form section, however each entry consists only of two values:
+
+#### Item parameters
+
+- **path** - _[required]_ contains the path to the data to use for the column.
+- **widget** - optional widget used to render the field. By default the value will be displayed verbatim. For more information about the available widgets see [Display widgets](display-widgets.md).
```

Comment:
We are missing information about the `name` property which can be used to specify the name of the column

Reply:
the name of the column is taken from i18n based on path